openforcefield/topology/topology.py

```diff
@@ -1438,6 +1438,54 @@ def from_dict(cls, d):
         # Implement abstract method Serializable.to_dict()
         raise NotImplementedError()  # TODO
 
+    @staticmethod
+    def _networkx_to_hill_formula(mol_graph):
```

Comment:
Added TODO in 9be939c 

I think we'll eventually make a `Molecule.from_networkx` function, but that's held up because we need to be careful about defining what we expect in a networkX representation of a molecule. When we do make that conversion function, I agree that we should move the hill formula-printing code into `Molecule`. I've said as much in the TODO. Closest related issue to tag here is #194 .